TestingExplainer/ViewController.swift

```diff
@@ -281,6 +284,8 @@ class OverlayViewController: UIViewController {
         
             let convertedFrame = self.view.convert(highlightView.frame, from: highlightView.superview)
             highlightBorderView?.frame = convertedFrame.insetBy(dx: -padding.width, dy: -padding.height)
+
+                                                    // I bet this is undocumented, isn't it?
```

Comment:
very much so, but this was a quick proof of concept. using our own bar button items we could enforce the use of custom view